tests/test_sensitivity_matrix.py

```diff
@@ -65,19 +65,67 @@ def setUp(self):
             # which we have excluded from the unrotated_sensitivity_matrix.
             self.gq_sensitivity_matrix = self.gq_sensitivity_matrix[:31, ...]
 
+            # Because the old sensitivity matrix was in the Zemax Coordinate
+            # System we need to swap the following zernikes:
+            swap = np.array([5, 8, 10, 13, 15, 16, 18, 20])
+            self.gq_sensitivity_matrix[:, swap - self.ofc_data.znmin] *= -1
+
+            # Similarly, we need to swap the following gaussian
+            # quadrature points because the old sensitivity matrix
+            # was in the Zemax Coordinate System and therefore
+            # there is a flip in the x direction. So the following
+            # swap, flips those GQ points that have opposite x sign.
```

Comment:
Same comment of the coordinate system above.

Reply:
Same as above